docs/index.md

```diff
@@ -130,11 +127,11 @@ if __name__ == '__main__':
 	main()
 ```
 #### `From Main` Explained:
-**Using the [Hydra](https://hydra.cc/docs/intro/){:target="_blank"} library to manage configuration for your script.**
+**Using the [Hydra](https://hydra.cc/docs/intro/){:target="_blank"} library to manage configuration for your Core-Lib library.**
 
 #### Decorator: @hydra.main:
-- This decorator tells `Hydra` to use the specified `YAML` configuration file (`core_lib_config.yaml`) located in the current directory ('.') to configure your application. It's a convenient way to manage configurations for your script.
-- **`def main(cfg)`**: This is the main function of your script, which takes a `cfg` argument. This argument will hold the configuration provided by `Hydra`.
+- This decorator tells `Hydra` to use the specified `YAML` configuration file (`core_lib_config.yaml`) located in the current directory ('.') to configure your application. It's a convenient way to manage configurations for your Core-Lib library.
```

Comment:
maybe better "located in the root directory of your core-lib project"

Reply:
remove "It’s a convenient way to manage configurations for your Core-Lib library."